README.md

```diff
@@ -1 +1,50 @@
-# Review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-Microverse
+
+This project is to demonstrate the ability to be a code reviewer
+
+## Built With
+- HTML
+- CSS
+
+## Live Demos
+
+[Live Demo Link](https://jbrime15.github.io/Review/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Webbrowser and access to internet
+
+### Install
+
+1) Clone the repository to your machine;
+
+```sh
+$ git clone https://github.com/jbrime15/Review.git
+```
+
+2) Navigate to the project folder and execute the "index.html" file.
+
+## Author
+
+👤 **JUBRIL IBRAHIM**
+
+- GitHub: [@jbrime15](https://github.com/jbrime15)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
```

Comment:
> Your Read.me File should describe the project

It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes 😎

Also, please be sure to tell your requirements/suggestions in a way that sounds like suggestions instead of orders 😉 The idea is to make it politely and friendly 😬